### DISCUSSION TITLE
Connection to MetaBrainz Foundation Inc.

### DISCUSSION THREAD

**Discussion by SinewaveLaboratorium** [Tyrrrz/YoutubeDownloader#336] [Q&A] [answered]
While downloading some videos, I noticed that YouTubeDownloader was connecting to some IPs belonging to Google via the 443 port, but also another extra connection to 138.201.227.205 (MetaBrainz Foundation) via the 80 port. What's the deal with that?

**Comment by Tyrrrz** [comment, marked as answer]
It's used for tagging music files: https://github.com/Tyrrrz/YoutubeDownloader/blob/master/YoutubeDownloader.Core/Tagging/MusicBrainzClient.cs
